### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jburel**
Add more python code check
and exclude directories generating when prep for pypi upload
Check that travis is green and that gallery is still working

### CODE REVIEW

**Review by atarkowska — CHANGES_REQUESTED** [review]
You have to update `urls.py` accordingly otherwise views throws an exception like

```
TypeError: dataset() takes at least 2 arguments (2 given)
```

**Comment by jburel** [comment]
@aleksandra-tarkowska did you have a chance to look at the changes?

**Comment by atarkowska** [comment]
Going through that amount of changes line by line and checking each character is not possible. We need tests for that. How do you think we could get tests in place.

As you are cleaning up the code, would you be ok to review `request.REQUEST`?

**Comment by jburel** [comment]
The "characters" changes are done by travis.
Using the app will check if the urls changes are correctly made.

File: omero_webtest/urls.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
     # View a dataset as two panels of images,
     # each with different rendering setting
-    url(r'^dataset_split_view/(?P<datasetId>[0-9]+)/',
```
Comment (atarkowska) [inline omero_webtest/urls.py:43]: still failing `TypeError: dataset_split_view() takes at least 2 arguments (2 given)`

**Review by atarkowska — COMMENTED** [review]
I went briefly through all the changes in urls.py. Noticed one issue

File: omero_webtest/urls.py (hunk 1 above)
Comment (jburel) [reply omero_webtest/urls.py:43]: are you sure you are running the latest code?
This was fixed by 4eba3c2

Comment (atarkowska) [reply omero_webtest/urls.py:43]: I am afraid the same behaviour http://web-dev-merge.openmicroscopy.org/webtest/dataset_split_view/7/ I can reproduce locally in isolation.

Comment (jburel) [reply omero_webtest/urls.py:43]: found the culprit ``dataset_split_include``

**Review by atarkowska — APPROVED** [review]

**Comment by jburel** [comment]
Tested today the "split_view_figure": no crash when clicking on save. Just the csrf message.
@aleksandra-tarkowska and I discussed the csrf issue. this will be reviewed in another PR

**Comment by atarkowska** [comment]
As discussed with @jburel `request.REQUEST` will be reviewed in another PR as well. My proposal is to write few simple tests like https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/test/integration/test_csrf.py that will allow to review both in one go

**Pull request merged**